app/controllers/events/list.js

```diff
@@ -15,10 +15,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         extraValuePaths : ['name', 'logoUrl'],
         headerComponent : 'tables/headers/sort',
         cellComponent   : 'ui-table/cell/cell-event-general',
-        options         : {
-          dateFormat: 'MMMM DD, YYYY - HH:mm A'
-        },
-        actions: {
```

Comment:
Same

Reply:
I will handle it in another PR.

Reply:
OK,then revert it

Reply:
@iamareebjamal Done, Please review and merge